tests/integration_tests/validation/run_conformance_suites.py

```diff
@@ -127,18 +127,24 @@ def run_conformance_suites(
         for conformance_suite_config in conformance_suite_configs:
             download_conformance_suite(conformance_suite_config, overwrite=overwrite)
     if download_cache:
-        package_urls = set()
+        package_paths = set()
         for conformance_suite_config in conformance_suite_configs:
-            package_urls.update(conformance_suite_config.package_urls)
+            for package_name, package_path in conformance_suite_config.package_paths.items():
+                download_taxonomy_package(package_name, package_path)
+                package_paths.add(package_path)
             if not conformance_suite_config.network_or_cache_required:
                 continue
             download_and_apply_cache(
                 f'conformance_suites/{conformance_suite_config.name}.zip',
                 version_id=conformance_suite_config.cache_version_id
             )
-        if package_urls:
+        if package_paths:
             # Download the packages.
-            subprocess.run([sys.executable, 'arelleCmdLine.py', '--packages', '|'.join(sorted(package_urls)), '--proxy', 'show'])
+            subprocess.run([
```

Comment:
Is this still needed to download packages, or are they already downloaded by the code above?